test/parallel/parallel_test_runner.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Get current test file name
+TESTFILE_NAME=${PGAPPNAME##pg_regress/}
+# Set different name for each test database
+# As pg_regress does not support cleaning db after each test
+TEST_CASE_DB="ldb_parallel"
+# Set database user
+if [ -z $DB_USER ]
+then
+     echo "ERROR: DB_USER environment variable is not set before test_runner.sh is run by pg_regress"
+     exit 1
+fi
+
+# Drop db after each test on exit signal
+function drop_db {
+  cat <<EOF | psql "$@" -U ${DB_USER} -d postgres -v ECHO=none >/dev/null 2>&1
+    SET client_min_messages=ERROR;
+    DROP DATABASE "${TEST_CASE_DB}";
+EOF
+}
+
+if [[ "$TESTFILE_NAME" =~ ^end ]]; then
+    trap drop_db EXIT
+fi
+
+
+# Change directory to sql so sql imports will work correctly
+cd sql/
+# install lantern extension
+if [[ "$TESTFILE_NAME" =~ ^begin ]]; then
+    psql "$@" -U ${DB_USER} -d postgres -v ECHO=none -q -c "CREATE DATABASE ${TEST_CASE_DB};" 2>/dev/null
+    psql "$@" -U ${DB_USER} -d ${TEST_CASE_DB} -v ECHO=none -q -c "SET client_min_messages=error; CREATE EXTENSION lantern;" 2>/dev/null
+    #psql "$@" -U ${DB_USER} -d ${TEST_CASE_DB} -v ECHO=none -q -f utils/common.sql 2>/dev/null
+fi
+
+# Exclude debug/inconsistent output from psql
```

Comment:
is it possible to create some `common_runner` file wrap this into a function and use that in both runners? Also if it's an option we can separate all the common parts in these scripts

Reply:
Ah yeah, I had initially split it out so I could test more easily, I'll combine these